index.html

```diff
@@ -1361,6 +1585,8 @@ <h2>
                 <li>[=Issue a haptic effect=] to the actuator with |type|,
                 |params|, and the |playEffectTimestamp|.
                 </li>
+                <li>If |this|'s [=gamepad=] is a [=simulated gamepad=], run the steps to [=vibrate a simulated gamepad=] for |this|'s [=gamepad=] with |params|.
+                </li>
```

Comment:
```suggestion
                <li>If |this|'s [=gamepad=] is a [=simulated gamepad=], run the steps to [=vibrate a simulated gamepad=] for |this|'s [=gamepad=] with |params|.
                </li>
                <li>Otherwise, [=issue a haptic effect=] to the actuator with |type|,
                |params|, and the |playEffectTimestamp|.
                </li>
```